jared_manning/DLists/main.py

```diff
@@ -0,0 +1,51 @@
+class Node:
+    def __init__(self, value):
+        self.value = value
+        self.prev = None
+        self.next = None
+
+    def __str__(self):
+        return f'Value: {self.value}, Prev: {self.prev}, Next: {self.next}'
+
+
+class DList:
+    def __init__(self):
+        self.head = None
+
+    def add_head(self, value):
+        if self.head is None:
+            node = Node(value)
+            node.prev = None
```

Comment:
not really required (line 18, 23) because the node class __init__ function takes care of it.